recipes/ukpopulation/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "ukpopulation" %}
+{% set version = "1.0.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/06/4e/bd7d79d83a0b5535dabfec2bc245bec1a30cf918c4dd2bba336a3c4d1a7c/{{ name }}-{{ version }}.tar.gz
+  sha256: 8c2320796285e011cf93ffcc14f822ffe6dc4102b8fa4ecc9b44a7b788442904
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - pandas
+    - requests
+    - openpyxl
+    - beautifulsoup4
+    - lxml
+    - ukcensusapi
+
+test:
+  imports:
+    - ukpopulation.myedata
+    - ukpopulation.snppdata
+    - ukpopulation.nppdata
+
+about:
+  home: https://github.com/nismod/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENCE
+  summary: Download, cache, collate, filter and extrapolate UK Population estimates and projections
+  dev_url: https://github.com/nismod/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - virgesmith
+    - tomalrussell
```

Comment:
@tomalrussell -- are you ok being added as a co-maintainer of the `ukpopulation` recipe on conda-forge?

https://conda-forge.org/docs/recipe.html#maintainer-role

Reply:
Yes, that's fine.